link_shorterner/test/shortener_shortener_SUITE.erl

```diff
@@ -0,0 +1,70 @@
+-module(shortener_shortener_SUITE).
+
+-include_lib("common_test/include/ct.hrl").
+
+-export([all/0,
+         init_per_testcase/2,
+         end_per_testcase/2]).
+-export([test_notfound/1,
+         test_created/1,
+         test_ok/1]).
+
+all() ->
+    [test_notfound,
+     test_created,
+     test_ok].
+
+init_per_testcase(_,Config) ->
+    start_web_server(),
+    Config.
+
+end_per_testcase(_, Config) ->
+    stop_web_server(),
+    Config.
+
+test_notfound(_) ->
+    lists:foreach(
+      fun(_) ->
+              {404, _} = do_get_request(gen_random_url())
+      end, lists:seq(1,10)).
+
+test_created(_) ->
+    lists:foreach(
+      fun(_) ->
+              {201, _} = do_post_request(gen_random_url())
+      end, lists:seq(1,10)).
+
+test_ok(_) ->
+    lists:foreach(
+      fun(_) ->
+              NewUrl = gen_random_url(),
+              {201, _} = do_post_request(NewUrl),
+              {200, _} = do_post_request(NewUrl)
+      end, lists:seq(1,10)).
+
+get_request_url(Url) -> 
+    BinaryReqUrl = iolist_to_binary([<<"http://localhost:8080/">>, Url]),
```

Comment:
BinaryReqUrl = <<"http://localhost:8080/", Url/binary>>

If you have all binaries is better to use binary constructors instead of iolilst (assuming Url is a binary here)

Reply:
Also `<<"http://localhost:8080/">>` is the kind of thing you want to put in the test Config and define it once in the init per suite